docs.html

```diff
@@ -663,6 +664,71 @@ <h2 class="component-example">Examples</h2>
           </div>
         </div>
         <hr />
+        <div id="modal">
+          <div class="component-intro">
+            <h1 class="component-title">Modal</h1>
+            <p>
+              Modals are built with HTML, CSS, and JavaScript. They’re
+              positioned over everything else in the document and remove scroll
+              from the body so that modal content scrolls instead.
+            </p>
+          </div>
+
+          <h2 class="component-example">Examples</h2>
+          Click on the below CTA to open modal.
+          <div
+            style="flex-direction: column; border: 1px solid gray"
+            class="button-examples"
+          >
+            <button style="width: 15%" id="myBtn">Click Me</button>
+
+            <div id="myModal" class="modal">
+              <div class="modal-content">
+                <span class="close">&times;</span>
+                <p>Modal body containing some text</p>
+                <img
+                  src="https://cdn.pixabay.com/photo/2015/04/23/22/00/tree-736885__480.jpg"
```

Comment:
here you have used empty **alt** attribute, provide some text to **alt** attribute